### DISCUSSION TITLE
Power-Fx running on Blazor WASM

### DISCUSSION THREAD

**Discussion by ryzam** [microsoft/Power-Fx#140] [Q&A] [answered]
HI Guys, any plan to create a demo project showing how to integrate Power-Fx with Blazor WASM. It's nice also to have power-fx-formulabar implemented as Blazor Component

**Comment by MikeStall** [comment]
We do plan to create a demo. We've done some experimenting and we're confident it's possible.

**Comment by hajekj** [comment, marked as answer]
Little late to the show, but here is a working WASM sample: https://networg.github.io/power-fx-wasm-demo/

Source is available here: https://github.com/NETWORG/power-fx-wasm-demo

We have some plans to make it broadly available to others via a CDN as a library which you can include in your code, stay tuned.

**Comment by MikeStall** [comment]
Here is another example: https://truegeek.com/2024/01/10/power-fx-blazor-demo/
